demo/pages/popover.py

```diff
@@ -0,0 +1,27 @@
+# popover.py
+import dash_material_ui as mui
+import dash
+from dash import html
+from util import create_component_item
+
+dash.register_page(__name__)
+
+button_popover_props = {
+    "id": "button-popover",
+    "content": "This is the content of the Popover opened by button click.",
+    "anchorOrigin": {"vertical": "bottom", "horizontal": "left"},
+}
+
+layout = html.Div(
+    mui.Grid(
```

Comment:
Same for this one: A lot of white space, and the code is cramp in the corner:
<img width="800" alt="image" src="https://github.com/user-attachments/assets/76700790-5d2c-4364-9f6f-7002313baa8d">